polyply/src/meta_molecule.py

```diff
@@ -172,6 +185,19 @@ def split_residue(self, split_strings):
         self.relabel_and_redo_res_graph(mapping)
         return mapping
 
+    @property
+    def search_tree(self):
+
+        if self.__search_tree is None:
+            if self.root is None:
+                self.root =_find_starting_node(self)
```

Comment:
Consider making `root` a propery with a getter and a setter.

Reply:
I made this into an issue